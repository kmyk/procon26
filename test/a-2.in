00101111111111111111111111111111
10000111111111111111111111111111
11110011111111111111111111111111
11111010001111111111111111111111
11111000111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111

3
11010000
01110000
00000000
00000000
00000000
00000000
00000000
00000000

10000000
11000000
01000000
01000000
00000000
00000000
00000000
00000000

01110000
11000000
00000000
00000000
00000000
00000000
00000000
00000000
