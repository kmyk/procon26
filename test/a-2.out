0 0 H 0
4 1 H 0
6 3 H 0
